deploy/setup.sh

```diff
@@ -12,4 +12,9 @@ echo \
    tee /etc/apt/sources.list.d/docker.list > /dev/null
 
 apt-get update -y
-apt-get install -y docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+
+# Manage Docker as a non-root user
+adduser  --disabled-password --gecos "" deployment
+usermod -aG docker deployment
+newgrp docker
```

Comment:
can we delete the ubuntu user after this?

Reply:
I decided to just use the ubuntu user (remove it from sudo group and add it to docker).